### PR TITLE
Redo Aave linear factory v2 deployment task

### PR DESCRIPTION
**Pull request by nventuro**
# Description

This deletes the old deployment task (which had not been used in any network) and replaces it with this updated one, which includes a fixed version string, the new event with protocol IDs, and configurable pause windows (though we don't use this).

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [x] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## Checklist:

- [x] The diff is legible and has no extraneous changes
- [x] Complex code has been commented, including external interfaces
- [x] Tests are included for all code paths
- [x] The base branch is either `master`, or there's a description of how to merge

### CODE REVIEW

File: pkg/deployments/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 - Deployed `L2GaugeCheckpointer` to Mainnet.
 - Deployed `VeBoostV2` to Mainnet.
 
+### Breaking Changes
+
+- The `20221115-aave-rebalanced-linear-pool` task was deleted and replace with `20221207-aave-rebalanced-linear-pool-v3`. The old task had not been used by anyone.
```
Comment (EndymionJkb) [inline pkg/deployments/CHANGELOG.md:12]:
```suggestion
- The `20221115-aave-rebalanced-linear-pool` task was deleted and replaced with `20221207-aave-rebalanced-linear-pool-v3`. The old task had not been used by anyone.
```

File: pkg/deployments/tasks/20221207-aave-rebalanced-linear-pool-v3/readme.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
-# 2022-11-15 - Aave Rebalanced Linear Pool
+# 2022-12-07 - Aave Rebalanced Linear Pool v3
 
 Deployment of the `AaveLinearPoolFactory`, for Linear Pools with a wrapped aToken. This new deployment includes:
 
 - A fix for the `AaveLinearPool` being susceptible to spoofed revert data from the external call to fetch the wrapped token rate, potentially resulting in manipulation of the exchange rate used by `AaveLinearPoolRebalancer`.
 - A fix for the `AaveLinearPoolRebalancer` to handle tokens which do not allow setting an approval without first setting the approval to zero.
```
Comment (EndymionJkb) [inline pkg/deployments/tasks/20221207-aave-rebalanced-linear-pool-v3/readme.md:6]: Also has versioning, the Protocol ID on create, and configurable pause window (though we are deploying it with standard pause durations)

Comment (EndymionJkb) [reply pkg/deployments/tasks/20221207-aave-rebalanced-linear-pool-v3/readme.md:6]: The new action ids should have one for `registerProtocolId`

**Review by jubeira — APPROVED** [review]
LGTM!
I've verified that the build info for the aave linear pool factory matches the one submitted in this PR.

**Review by EndymionJkb — APPROVED** [review]
Should be fine to delete the old one if we're sure nobody used it; don't want to clutter /deprecated unnecessarily. Just want to make sure we're documenting all the new things.

**Pull request merged**